Sources/Data/PhotoStorage.swift

```diff
@@ -62,12 +62,11 @@ public func ==(lhs: Asset, rhs: Asset) -> Bool {
 
 // Public API for creating Asset's
 open class PhotoStorage {
-    fileprivate let baseURL: URL
-    fileprivate let queue = DispatchQueue(label: "no.finn.finjonon.disk-cache-writes", attributes: [])
-    fileprivate let resizeQueue = DispatchQueue(label: "no.finn.finjonon.disk-cache-resizes", attributes: DispatchQueue.Attributes.concurrent)
-    fileprivate let fileManager = FileManager()
-    fileprivate var cache: [String: Asset] = [:]
-    fileprivate let assetLibrary = ALAssetsLibrary()
+    private let baseURL: URL
+    private let queue = DispatchQueue(label: "no.finn.finjonon.disk-cache-writes", attributes: [])
+    private let resizeQueue = DispatchQueue(label: "no.finn.finjonon.disk-cache-resizes", attributes: DispatchQueue.Attributes.concurrent)
```

Comment:
Could this be refactored to `DispatchQueue(label: "no.finn.finjonon.disk-cache-resizes", attributes: .concurrent)` instead?